### PR TITLE
Add API keys

### PR DESCRIPTION
**Pull request by devkiran**

### CODE REVIEW

File: pages/api/teams/[slug]/api-keys/[apiKeyId].ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  const session = await getSession(req, res);
+
+  if (!session) {
+    throw new Error('Unauthorized');
+  }
+
+  const { slug, apiKeyId } = req.query as { slug: string; apiKeyId: string };
+
+  if (!(await hasTeamAccess({ userId: session.user.id, teamSlug: slug }))) {
+    throw new Error('You are not allowed to perform this action');
```
Comment (niwsa) [inline pages/api/teams/[slug]/api-keys/[apiKeyId].ts:41]: Could send 403 response here.

Comment (devkiran) [reply pages/api/teams/[slug]/api-keys/[apiKeyId].ts]: Done

File: components/shared/InputWithCopyButton.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      <div className="flex justify-between items-center">
+        <label className="label">
+          <span className="label-text">{label}</span>
+        </label>
+        <CopyToClipboardButton value={value?.toString() || ''} />
+      </div>
+      <Input
+        className="input input-bordered w-full"
+        {...rest}
+        value={value}
```
Comment (niwsa) [inline components/shared/InputWithCopyButton.tsx:24]: Should use `defaultValue` prop instead of `value`

**Review by niwsa — APPROVED** [review]

**Comment by deepakprabhakara** [comment]
@devkiran @niwsa Whenever prisma schema changes you should generate migration scripts

**Pull request merged**